prisma/migrations/20240711175230_add_user_timestamps_and_rename_fields/migration.sql

```diff
@@ -0,0 +1,23 @@
+/*
+  Warnings:
+
+  - You are about to drop the `User` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropTable
+DROP TABLE "User";
+
+-- CreateTable
+CREATE TABLE "users" (
+    "id" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+    "email" TEXT NOT NULL,
+    "hash" TEXT NOT NULL,
+    "hashedRt" TEXT,
```

Comment:
consider renaming `hash` to `password` or `hashedPassword` and `hashedRt` to `refreshToken` or `hashedRefreshToken`

Reply:
also consider our discussion on slack about hashing refresh token 🙂